lib/stages/testDataGen.cpp

```diff
@@ -209,4 +209,5 @@ void testDataGen::main_thread() {
                 usleep((int)(1e6 * (frame_end_time - time)));
         }
     }
+    INFO("Calling dump_trackers");
```

Comment:
Might not need this message?

Reply:
oh yeah, this is a testing log message.  Will remove.